coptic_char_generator.py

```diff
@@ -234,18 +234,38 @@ def train_model(model, train_data, dev_data=None, output_name="charLM"):
         )
 
         logging.info(f"orig sentence: ⲙ̅ⲡϥ̅ⲟⲩⲱϣⲉϭⲱ̅ϣⲁⲁⲧⲉⲡⲣⲟⲑⲉⲥⲙⲓⲁⲙ̅ⲡⲉϥⲁϩⲉ·")
-        seed = "ⲙ̅ⲡϥ̅ⲟⲩ"
-        sample = generate(model, seed=seed, n=200, temp=0)
+        seed = "ⲙ̅ⲡϥ̅ⲟⲩⲱϣ<mask>ϭⲱ̅ϣⲁⲁⲧⲉⲡⲣⲟ<mask>ⲉⲥⲙⲓⲁⲙ̅ⲡⲉϥⲁϩⲉ·"
+        logging.info(f"masked sentence: {seed}")
+
+        sample = fill_masks(model, seed, temp=0)
         logging.info(f" genertated output: {sample}")
 
         torch.save(model, f"{model_path}/{output_name}.pth")
 
     return model
 
 
-def generate(model, seed="The ", n=150, temp=0):
-    # TODO update this to find <mask> tokens and fill them in instead of generating text out
+def fill_masks(model, text, temp=0):
```

Comment:
turned your code from earlier into this function instead, just took out the labels b/c they aren't in lookup_indexes anymore